frontend/lib/identification/card_detail_view/card_detail_view.dart

```diff
@@ -157,6 +160,8 @@ enum CardStatus {
       return CardStatus.timeOutOfSync;
     } else if (!code.cardVerification.cardValid) {
       return CardStatus.invalid;
+    } else if (isCardNotYetValid(code.info)) {
+      return CardStatus.notYetValid;
     } else {
```

Comment:
Sorry, but one more issue I think:

Shouldn't these two cases be swapped?
If the card is not yet valid, then the server would respond with `invalid`, thus the isCardNotYetValid case would (usually) never occur.



```suggestion
    } else if (isCardNotYetValid(code.info)) {
      return CardStatus.notYetValid;
    } else if (!code.cardVerification.cardValid) {
      return CardStatus.invalid;
    } else {
```